source/community/get-involved/donate-hardware.md

```diff
@@ -1,9 +1,21 @@
 ---
 title: Donate hardware
 category: develop
-authors: ederevea
+authors: ederevea sandrobonazzola
 ---
 
+# Resources donated so far
+
+| Quantity / Model   | Year | Donor   | warranty status              | Used for                      |
+|:---------------------------------------------------------------------------------------------------|
+| 2x PowerEdge R720  | 2014 | Red Hat | extended warranty active     | storage servers               |
+| 14x PowerEdge R620 | 2014 | Red Hat | **warranty expired in 2017** | hypervisors & OpenShift nodes |
+| 10x PowerEdge R430 | 2016 | Red Hat | **warranty expired in 2019** | OST workers                   |
+| 2x Power S812L     | 2015 | IBM     | *on loan from IBM*           | POWER8 hypervisors            |
+| 3x s390 VM         |      | IBM     | -                            | s390 hypervisors              | 
```

Comment:
Minor correction - s390 VMs are not used as hypervisors. They are directly connected to Jenkins and run for VDSM CI jobs for the platform. We can replace the description with `s390x CI workers`